### PR TITLE
Skip CI Test

### PR DESCRIPTION
**Pull request by jennyli-z**
Signed-off-by: Jenny Li <jing.li@zilliz.com>

### CODE REVIEW

**Review by jennyli-z — COMMENTED** [review]
/run-checks

**Comment by jennyli-z** [comment]
/? workflow code-checker.yaml

**Comment by jennyli-z** [comment]
/? help

**Comment by jennyli-z** [comment]
/? workflow code-checker.yaml

**Comment by jennyli-z** [comment]
/? help

File: internal/test/a/a.go [hunk 1]
```diff
@@ -3,4 +3,5 @@ package a
 
 const (
 	TestA = "testA"
+	TestB = "testB"
```
Comment (carawaylj) [inline internal/test/a/a.go:6]: /add-invalid

File: internal/test/a/a.go [hunk 2]
```diff
@@ -3,4 +3,5 @@ package a
 
 const (
 	TestA = "testA"
```
Comment (carawaylj) [inline internal/test/a/a.go:5]: /run-checks

File: internal/test/a/a.go [hunk 3]
```diff
@@ -3,4 +3,5 @@ package a
 
 const (
```
Comment (carawaylj) [inline internal/test/a/a.go:4]: /run-checks

**Review by carawaylj — COMMENTED** [review]
/run-checks

**Review by carawaylj — COMMENTED** [review]
test

**Review by carawaylj — COMMENTED** [review]
/run-checks

**Pull request closed without merging**